spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/EnvironmentEntryInitializingTreeMap.java

```diff
@@ -52,18 +52,18 @@
 	/**
 	 * Constructs the map.
 	 * 
-	 * @param applicationContext the application context that supplies the property values
+	 * @param environment the environment that supplies the default property values
 	 * @param entryClass the entry class
 	 * @param defaultsPrefix the prefix for initializing the properties
 	 * @param delegate the actual map that stores the values
 	 */
-	public EnvironmentEntryInitializingTreeMap(ConfigurableApplicationContext applicationContext, Class<T> entryClass,
+	public EnvironmentEntryInitializingTreeMap(ConfigurableEnvironment environment, Class<T> entryClass,
 			String defaultsPrefix, Map<String, T> delegate) {
```

Comment:
Not specific to this PR; but can we make the `defaultsPrefix` to be a constant (with the value `spring.cloud.stream.default`) in this class instead of passing from the invoking method?